releasenotes/notes/gateway-target-port.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: networking
+issue:
+  - 29291
+
+releaseNotes:
+- |
+  **Fixed** an issue where duplicated listeners are generated when Gateway server ports use mix of service port and target port.
```

Comment:
Maybe **Fixed** an issue where duplicated listeners are generated when Gateway server ports use **a** mix of service target ports

Reply:
The issue happens if user have two gateways, one points to service port (say 80), the other one points to the target port of that service port (say 8080). So I think the current way to describe the issue is more accurate?